integration-tests/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "newman run checkmate.postman_collection.json -g ./env.json",
+    "test": "newman run checkmate.postman_collection.json -g ./env.json --delay-request 2000",
```

Comment:
This seems to affect every request, however only some requests require delay. The test suite with this runs in 12 mins vs 7 mins.
Did you happen to add many new tests (`integration-tests/checkmate.postman_collection.json` diff too large to view) that might increase this timing? Or is it perhaps from the delays?

Reply:
it's from the delays! haha I was lazy to individually add delays to those that are async. Is there a better way?

Reply:
Nah. Just leave it for now. We can figure out how to optimise when it reaches 1hr. Lol